tests/cases/fourslash/pushTypeGetTypeOfAlias.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path='fourslash.ts'/>
+// @allowJs: true
+
+// @Filename: bar.js
+//// module.exports = function () {};
+//// exports.blah = exports.unknown;
+
+// @Filename: foo.js
+//// var bar = require("./bar");
+//// bar./**/
+
+verify.baselineCompletions();

```

Comment:
You can repro this in tsc without a fourslash test, so I would just switch to that.

```ts
// @checkJs: true

// @filename: a.js
module.exports = function () { };
exports.blah = exports.unknown;

// @filename: b.js
var bar = require("./a.js");
bar.asd;
```